extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/include.config.html

```diff
@@ -86,10 +86,16 @@
 									</p>
 								</span>
 							</div>
-							<span ng-show="focus">
-								<p class="hint">
-									<small>&nbsp;</small>
-								</p>
+							<span>
+								<md-chips class="scriptChips" id="{{parameter.name}}" ng-show="items && items.length>0" ng-model="items" placeholder="Add an item" md-on-select="selectChip($chip,parameter.name)">
+								<input disabled/>
+								 <md-chip-template> <strong>{{$chip.name}}</strong>
+								 </md-chip-template> </md-chips>
+								<span>
+                                    <p class="hint noMargin">
+                                        <small>available outputs</small>
+                                    </p>
+                                </span>
```

Comment:
Not broken. My eclipse does not format html files on save. I format them manually.